scripts/generate-schemas.js

```diff
@@ -0,0 +1,33 @@
+const { exec } = require('child_process')
+const fs = require('fs')
+const { promisify } = require('util')
+
+SCHEMA_PATH = 'libs/api/schema/src/lib/schema.d.ts'
+
+YARN_COMMANDS = [
+  'yarn nx run application-system-api:build-schema',
+  'yarn nx run api-domains-application:codegen',
+  'yarn nx run api:build-schema',
+  'yarn affected:schemas',
+]
+
+const main = async () => {
+  if (!(await promisify(fs.exists)(SCHEMA_PATH))) {
+    await promisify(fs.writeFile)(SCHEMA_PATH, 'export default () => {}')
+  }
+  for (cmd of YARN_COMMANDS) {
```

Comment:
Better to use `forEach/map/reduce` (in this case, `forEach`). These looping constructs treat empty elements differently.

Reply:
After DM we decided not to do that